galgebra/ga.py

```diff
@@ -25,9 +25,10 @@
 )
 from ._utils import cached_property as _cached_property
 
-half = Rational(1, 2)
-one = S(1)
-zero = S(0)
+# This file does not and should not use these.
+# Unfortunately, some of our examples do.
+one = S.One
+zero = S.Zero
```

Comment:
We should keep `half` or remove all of them, there should be no middle ground.

Reply:
I thought I'd just remove `half` of them

Reply:
All the others are used by the examples and tests, and it's rather involved to change them. I can add back `half` anyway if you'd like.

Reply:
> I can add back `half` anyway if you'd like.

Thanks.

Is it possible to also deprecate them? I don't know if it works for variables......

Reply:
It's difficult enough that I didn't want to do it at the same time.

Reply:
:)

Reply:
Added back this `half`.